spec/extensions/forbid_lazy_load_spec.rb

```diff
@@ -15,312 +16,364 @@
     @o2 = @c.load(:id=>2, :t_id=>3)
   end
 
-  it "should not forbid lazy load if not set at instance level" do
-    @o1.t.must_equal @o2
-    @o1.ts.must_equal [@o2]
-    @o1.mtm_ts.must_equal [@o2]
-    @o1.otoo_t.must_equal @o2
-    @o1.oto_t.must_equal @o2
-  end
+  model_instance_specs = Module.new do
```

Comment:
I extracted the core specs for model instances into this set and then included them in the two groups below.